projetos/axios-api-request/style.css

```
@@ -0,0 +1,107 @@
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans+Mono:wght@100;200;500;700;800&display=swap');
+body {
```

Sempre adicionar um espaço entre cada tag do CSS 💅🏻 

```suggestion

body {
```